docs/tutorial/tutorial-5/index.rst

```diff
@@ -32,29 +32,15 @@ So far, we've been running and testing our application on the desktop. However,
 BeeWare also supports mobile platforms - and the application we've written
 can be deployed to your mobile device, too!
 
-.. rst-class:: clearfix row
-
-.. rst-class:: column column2
-
 :doc:`iOS <iOS>`
 ----------------
 
-iOS applications can only be compiled on macOS. You'll need `Xcode
-<https://apps.apple.com/au/app/xcode/id497799835?mt=12>`__, which you should
-have installed in :doc:`Tutorial 0 <../tutorial-0>`.
-
-Once you've got Xcode installed, :doc:`let's build our app for iOS! <iOS>`
```

Comment:
I think it makes more sense to put the tool installation instructions at the top of the iOS and Android sub-pages.

Reply:
Agreed - but we should retain a call to action directing them to the next page. Yes, it should be obvious, but it never hurts to be explicit.